setup.json

```diff
@@ -20,18 +20,6 @@
             "cls": "SchedulerApp",
             "show": true,
             "pos": "bottom"
-        },
```

Comment:
How about making the `logger` located at the bottom? I think the logger must wrap all the others up.

Reply:
Good point. 
Since it determines overall design, I want to know how @kangz12345 and @BECATRUE think about this.
I will update it after their review.

Reply:
Well, I think changing the item arrangement in the json file does not matter. It seems okay to locate the "logger" part at the bottom of the json file.